packages/manager/src/factories/dashboards.ts

```diff
@@ -62,25 +63,41 @@ export const dashboardMetricFactory =
       (i) => scrape_interval[i % scrape_interval.length]
     ),
     unit: 'defaultUnit',
-  });
+  }
+);
 
-export const cloudPulseMetricsResponseDataFactory =
-  Factory.Sync.makeFactory<CloudPulseMetricsResponseData>({
+export const cloudPulseMetricsResponseDataFactory = Factory.Sync.makeFactory<CloudPulseMetricsResponseData>(
+  {
     result: [
       {
         metric: {},
         values: [],
       },
     ],
     result_type: 'matrix',
-  });
+  }
+);
 
-export const cloudPulseMetricsResponseFactory =
-  Factory.Sync.makeFactory<CloudPulseMetricsResponse>({
+export const cloudPulseMetricsResponseFactory = Factory.Sync.makeFactory<CloudPulseMetricsResponse>(
+  {
     data: cloudPulseMetricsResponseDataFactory.build(),
     isPartial: false,
     stats: {
       series_fetched: 2,
     },
     status: 'success',
-  });
+  }
+);
+
+export const userPreferencesFactory = Factory.Sync.makeFactory<UserPreferences>(
```

Comment:
do we need this in this file, since this is a factory for userPreferences, we should keep there accordingly